src/frontends/pytorch/src/utils_quantize.cpp

```diff
@@ -0,0 +1,243 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "utils_quantize.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/round.hpp"
+#include "openvino/op/minimum.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/fake_quantize.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/scatter_elements_update.hpp"
+#include "openvino/op/reshape.hpp"
+
+
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+
+using namespace ov::op;
+
+
+std::shared_ptr<ov::Node> quantize(const NodeContext& context,
+                                   std::shared_ptr<ov::Node> input,
+                                   std::shared_ptr<ov::Node> scale,
+                                   std::shared_ptr<ov::Node> zero_point,
+                                   ov::element::Type dtype,
+                                   QuantizedPtNodeType quantization_type,
+                                   std::shared_ptr<ov::Node> axis) {
+    if (quantization_type == QuantizedPtNodeType::QUANTIZE_PER_TENSOR) {
+        const auto input_convert = context.mark_node(std::make_shared<v0::Convert>(input, element::f32));
+        const auto scale_convert = context.mark_node(std::make_shared<v0::Convert>(scale, element::f32));
+        const auto zero_point_convert = context.mark_node(std::make_shared<v0::Convert>(zero_point, element::f32));
+
+        int64_t out_low_i64, out_high_i64;
+        if (dtype == element::u8) {
+            out_low_i64 = (int64_t)std::numeric_limits<unsigned char>::lowest();
+            out_high_i64 = (int64_t)std::numeric_limits<unsigned char>::max();
+        } else if (dtype == element::i8) {
+            out_low_i64 = (int64_t)std::numeric_limits<char>::lowest();
+            out_high_i64 = (int64_t)std::numeric_limits<char>::max();
+        } else if (dtype == element::f16) {
```

Comment:
I don't think `FakeQuantize` approach will work with fp16 data type. int32 is also some unusual case, lets only do int8/uint8 for now

Reply:
Removed fp16, int32 support left but removed from the tests with a note!